yt-to-webpage.pl

```diff
@@ -5,18 +5,23 @@
 use autodie;
 my $slug = shift @ARGV; 
 my $url = shift @ARGV;
+my $lang = shift @ARGV;
```

Comment:
you'd want `shift @ARGV || 'en';` here to do the defaulting.

Then you can get rid of the if (!$lang) bit below.

(I'd also recommend updating the usage further on to be [lang] rather than 'language' to make it clearer that's optional.)